drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -138,6 +141,57 @@ class CMMNotificationClient : public IMMNotificationClient {
 	}
 };
 
+void AudioDriverWASAPI::AudioDeviceWASAPI::Resampler::prepare(int p_input_mix_rate, int p_output_mix_rate, int p_channels) {
+	print_verbose("WASAPI: preparing to resample " + itos(p_input_mix_rate) + "Hz to " + itos(p_output_mix_rate) + "Hz");
+	input_mix_rate = p_input_mix_rate;
+	output_mix_rate = p_output_mix_rate;
+	channels = p_channels;
+	mix_offset = 0;
+	mix_increment = uint64_t(((input_mix_rate) / double(output_mix_rate)) * double(FP_LEN));
+
+	internal_buffer.resize(channels * (INTERNAL_BUFFER_LEN + CUBIC_INTERP_HISTORY));
+	auto internal_buffer_data = internal_buffer.ptrw();
```

Comment:
We don't allow using `auto` in Godot's codebase. Please write the type explicitly to make online code reviews easier.